src/ray/common/scheduling/cluster_resource_scheduler.cc

```diff
@@ -629,7 +686,7 @@ int64_t ClusterResourceScheduler::NumNodes() { return nodes_.size(); }
 
 void ClusterResourceScheduler::UpdateResourceCapacity(const std::string &client_id_string,
                                                       const std::string &resource_name,
-                                                      int64_t resource_total) {
+                                                      double resource_total) {
```

Comment:
should make this `FixedPoint`

Reply:
Actually the goal her was to not change the API in node_manager.cc

Then we need to do the mapping between double and FixedPoint in node_manager.cc.